libsysbox/syscont/spec.go

```diff
@@ -650,17 +651,92 @@ func cfgSystemdMounts(spec *specs.Spec) {
 	spec.Mounts = append(spec.Mounts, sysboxSystemdMounts...)
 }
 
-// sysMgrSetupMounts requests the sysbox-mgr to setup special sys container mounts.
-func sysMgrSetupMounts(mgr *sysbox.Mgr, spec *specs.Spec, uidShiftRootfs bool) error {
+// Obtains the docker data-root path utilized by the inner docker process to store its
+// data. This is used to define the container mountpoint on which the host's docker
+// volume (backing this resource) will be mounted on.
+//
+// Notice that even though this code habilitates the custom definition of the data-root's
+// location, this will be only honored by Sysbox if this attribute is set prior to the
+// container creation (i.e., at docker-image build time).
+func getDockerDataRootPath(spec *specs.Spec) string {
```

Comment:
Let's call this `getInnerDockerDataRootPath()` to avoid confusion with the host's docker.